server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "preact-server",
+	"amdName": "preactServer",
+	"version": "5.1.20",
```

Comment:
are we publishing this separately or are we going to keep this at its own version

Reply:
I think we can safely ignore this as the plan is to publish this as part of core and not as a separate package. The other addons have bogus version numbers to which don't match the same package. So far we haven't gotten an issue for that so I'm assuming that no tool reads it.